.github/workflows/test.yml

```diff
@@ -69,3 +69,14 @@ jobs:
           export PATH=$PATH:$HOME/.foundry/bin
           export PATH="$PATH:$GOPATH/bin"
           ./scripts/local/e2e_test.sh
+
+      - name: Upload tmpnet network dir
+        uses: actions/upload-artifact@v4
+        if: always()
+        with:
+          name: e2e-tmpnet-data
+          path: |
+            ~/.tmpnet/networks
+            ~/.tmpnet/prometheus/prometheus.log
+            ~/.tmpnet/promtail/promtail.log
+          if-no-files-found: error
```

Comment:
the prometheus stuff I just copied along with the rest of it from avalanchego 😅 but the `~/.tmpnet/networks` part is the most pertinent: that folder contains all of the logs, configs, etc for the network and all the nodes started for it. tmpnet chooses not to dump all of those logs to stdout, instead putting them on disk in that folder (a different sub-folder per tmpnet run). uploading that folder allows us to see the full diagnostic info for a test run, since it's not all included in the direct output of the CI job.

Reply:
Hmm... Would it make sense to tail the log file with a goroutine so the test suite outputs the output to stdout?

Reply:
maybe? this environment is currently spinning up 15 nodes, so it'd be tailing 15 different log files, which would be a lot of output. I guess it's no more than before, but Maru told me that the tmpnet philosophy is to dump log files to disk rather than flood stdout.

as an alternative, I used ginkgo's `AddReportEntry` function to inform the user of the directory where the the node logs can be found, in https://github.com/ava-labs/teleporter/pull/401/commits/956420b13cc5c5ca3768f97d082d37f6004082ff. what do you think of that?